src/core.js

```diff
@@ -118,7 +118,6 @@ extend( QUnit, {
 
 		// Initialize the configuration options
 		extend( config, {
-			stats: { all: 0, bad: 0, testCount: 0 },
```

Comment:
Uncaught errors now increment these. As errors can happen before/during `begin()`, I had to move this to `config.js` as the increment would fail otherwise with the stats object being undefined. We can move more of this over, but that's for another day.